invenio_indexer/api.py

```diff
@@ -69,27 +76,43 @@ def publish(self, data, **kwargs):
 
 
 class RecordIndexer(object):
-    """Record indexer.
-
-    Provides an interface for indexing records in Elasticsearch. Bulk indexing
-    works by queuing requests for indexing records and processing these
-    requests in bulk.
-
-    Elasticsearch index and doc_type for a record is determined from the
-    ``$schema`` attribute.
-
-    :param search_client: Elasticsearch client. Defaults to
-        ``current_search_client``
-    :param exchange: ``kombu.Exchange`` instance for message queue.
-    :param queue: ``kombu.Queue`` instance for message queue.
-    :param routing_key: Routing key for message queue.
-    :param version_type: Elasticsearch version type. Defaults to
-        ``external_gte``.
+    r"""Provide an interface for indexing records in Elasticsearch.
+
+    Bulk indexing works by queuing requests for indexing records and processing
+    these requests in bulk.
+
+    Example of bulk indexing:
+
+    .. code-block:: python
```

Comment:
@hachreak if we present python examples in the docstrings they should be runnable via doctests.